onnxruntime/core/providers/coreml/builders/impl/base_op_builder.cc

```diff
@@ -17,9 +17,10 @@ namespace coreml {
 // filter suppported ones.
 static std::set<std::string> Float16Ops = {
     "Add", "Mul", "Sub", "Div", "Pow", "Sqrt", "Reciprocal",
-    "Sigmoid", "Tanh", "Relu", "LeakyRelu", "Concat", "GridSample", "GlobalAveragePool",
```

Comment:
can these be listed one op per line and sorted alphabetically so it is easier to see what changes?

Reply:
Sorted by alphabetic already.

This list would be removed once all ops support ml program